src/btree/bt_page.c

```diff
@@ -455,13 +461,14 @@ __page_reconstruct_leaf_delta(WT_SESSION_IMPL *session, WT_REF *ref, WT_ITEM *de
                 tombstone->next = standard_value;
                 upd = tombstone;
             } else {
-                if (WT_TIME_WINDOW_HAS_START_PREPARE(&unpack.tw)) {
+                if (WT_TIME_WINDOW_HAS_START_PREPARE(&unpack.delta_value.tw)) {
```

Comment:
Why we need this here?

Reply:
Fixed [here](https://github.com/wiredtiger/wiredtiger/pull/12223/commits/94e17edc6daf629350f3adf776eab8b8e43ce5d3)